src/EventList.js

```diff
@@ -75,6 +75,11 @@ export function EventList(props) {
       }
     }
 
+    // ??? frm: React is complaining about there not being a unique key.  Here is the warning:
```

Comment:
I took a shot at this but have not figured it out yet.  I added some more comments in the code about the issue and I plan to return to it soon.  Don't want to hold up the merge for this though (since I have not changed any behavior - I have just added some comments).